test1/indexblog.html

```diff
@@ -0,0 +1,264 @@
+            <button>#this-website</button>
+        </div>
+    </div>
+    <div class="card-v7">
+        <h6> 1 May 2019</h6>
+        <div class="card-u7">
+            <h4>Django and CircleCI 2.0</h4>
+            <p>Most of the time, development sucks.Using a continuous integration app 
+                can help automate your deployments so you can focus on writing code 
+                and buildings things. In this post I'll show you how to set up a basic
+                Django project with CircleCI so that publishing your code is as simple
+                as pushing to GitHub.
+            
+            </p>
+            <button>#django</button>
+            <button>3circleCI</button>
+        </div>
+    </div>
+    <div class="card-v8">
+        <h6>20 Apr 2018</h6>
+        <div class="card-u8">
+            <h4>Ember Ajax and Ember Simple Auth</h4>
+            <p>It can be tricky to properly configure 'ember-ajax' to play 
+                nicely with 'ember-simple-auth'. You wwill need to manually 
+                configure a service to mimic the authorisation functionality you get 
+                for free when using Ember Simple Auth with Ember Data. This post 
+                will show you how to get up and running properly.
+            </p>
+            <button>#ember.js</button>
+        </div>
+    </div>
+    <div class="card-v9">
+        <h6>6 Apr 2018</h6>
+        <div class="card-u8">
+            <h4>Sending Ember actions down to components</h4>
+            <p>"Data down, actions up" is an often repeated mantra of the Ember
+                community, and for good reasons. Some-times though, you need to call
+                actions on components in a "downward" direction. In this post I'll give
+                you an approach for breaking this DDAU rule.
+            </p>
+            <button>#ember.js</button>
+        </div>  
+    </div>
+    <div class="card-v10">
+        <h6>13 July 2016</h6>
+        <div class="card-u10">
+            <h4>"Route variable" pattern with Ember</h4>
+            <p>There arre a couple of different libraries for Ember that 
+                add the ability to set the page title for each page in Your
+                application by simply creating a 'title' property on each route.
+                This is a great general purpose patterrn for declarative information
+                that needs to be set across routes. For example you mighht want
+                to add breadcrumbs, or just a different heading on each page. This Post
+                 will outline how to go about creating this pattern yourself. 
+            </p>
+            <button>#ember.js</button>  
+        </div>
+    </div>
+    <div class="sectionhead">
+        <p>1</p>
+        <p>2</p>
+        <p>3</p>
+        <p>4</p>
+        <p>&#8594;</p>
+    </div>
+</div>
+
+        <div class="footer">
+            <div class="footer_detail">
```

Comment:
Either please stick to `snake-case` for CSS class names or follow [BEM convention](https://getbem.com/introduction/)